service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessService.java

```diff
@@ -182,12 +182,11 @@ public AzureSasBundle createAzureStorageContainerSasToken(
       SasTokenOptions sasTokenOptions) {
     features.azureEnabledCheck();
 
-    var userEmail =
-        SamRethrow.onInterrupted(
-            () -> samService.getUserEmailFromSam(userRequest), "getUserEmailFromSam");
+    var samUser = samService.getSamUser(userRequest);
     logger.info(
-        "User {} requesting SAS token for Azure storage container {} in workspace {}",
-        userEmail,
+        "User {} [sub={}] requesting SAS token for Azure storage container {} in workspace {}",
```

Comment:
I would have no idea what "sub" is. Could you make it `[SubjectId={}]`?

Reply:
Done